app/Model/Server.php

```diff
@@ -7135,6 +7136,37 @@ private function generateServerSettings()
                 'type' => 'boolean',
                 'null' => true
             ),
+            'LinOTPAuth' => array(
+                'branch' => 1,
+                'baseUrl' => array(
+                    'level' => 2,
+                    'description' => __('The default LinOTP URL.'),
+                    'value' => 'https://linotp',
```

Comment:
Can this be abused if a default value without domain name is controlled by an attacker locally? Just wondering if it's better to keep it empty by default

Reply:
What would you think about:

`'value' => 'https://<your-linotp-baseUrl>',` ?

Reply:
Perfect thanks!